doc/source/whatsnew/v1.0.0.rst

```diff
@@ -490,7 +490,7 @@ Deprecations
 - :func:`eval` keyword argument "truediv" is deprecated and will be removed in a future version (:issue:`29812`)
 - :meth:`Categorical.take_nd` is deprecated, use :meth:`Categorical.take` instead (:issue:`27745`)
 - The parameter ``numeric_only`` of :meth:`Categorical.min` and :meth:`Categorical.max` is deprecated and replaced with ``skipna`` (:issue:`25303`)
--
+- :module:`pandas.core.index` has been deprecated and will be removed in a future version, use :module:`pandas.core.indexes.api` instead (:issue:`19711`)
```

Comment:
This should not be the recommended alternative, I think? (nothing in `pandas.core` should be public)

I think @jreback's suggestion to use `pandas.core.indexes` was for internal replacement

Reply:
so remove all of this note or just the part about using indexes.api?

Reply:
the docbuild doesnt like ":module:" here, what is the appropriate label?

Reply:
Nothing, just use double backticks for code. The `:obj:` things are only to link to something (like an API page), but for a module we have nothing to link to.

Reply:
> so remove all of this note or just the part about using indexes.api?

You can mention that all public classes (eg the several index classes) are available from the top-level namespace.

Reply:
thanks